stac_generator/raster_data/generator.py

```diff
@@ -0,0 +1,171 @@
+import csv
+import json
+import logging
+from datetime import UTC
+from typing import Any
+
+import pandas as pd
+import pystac
+import rasterio
+from pystac.extensions.eo import Band, EOExtension
+from pystac.extensions.projection import ProjectionExtension
+from pystac.extensions.raster import AssetRasterExtension, DataType, RasterBand
+from shapely.geometry import Polygon, mapping
+
+from stac_generator.base.generator import StacGenerator
+from stac_generator.base.schema import StacCatalogConfig, StacCollectionConfig
+
+from .schema import RasterSourceConfig
+
+# In case the config file has different names for bands
+BAND_MAPPING: dict[str, str] = {
```

Comment:
I am not sure what's the purpose of this since it seems to map to the same thing

Reply:
I think its fine to have the mapping dictionary prepared for the future.